### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by mestcihazal**
removed the repeated line, and changed few others for readability and consistency

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -36,8 +36,6 @@ For example: the following configuration would use the `ssd` framework:
 }
 ```
 
-Note: if you are going to use *get_detections_from_camera()*, you will need to set any camera that you plan to use as a dependency for this service.
```
Comment (mcvella) [inline README.md:39]: Why is this removed?  Isn't it still true?

**Review by mcvella — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (mestcihazal) [reply README.md:39]: It repeated a line before this, so you say it twice

**Review by mcvella — APPROVED** [review]
